terraform/modules/vpc/variables.tf

```diff
@@ -3,6 +3,8 @@ variable "name" {
   type        = string
 }
 
+variable "loadbalancer" {}
```

Comment:
can you add a description and type please?

Reply:
Is this variable being used? I can see it being passed in, but not used

Reply:
it was...Matt's found a way not to, so it can go AFAIK.  @mattburgess ?